AzureMonitorAgent/shim.sh

```diff
@@ -19,6 +19,10 @@ function find_python() {
     elif command -v python2 >/dev/null 2>&1 ; then
         eval ${python_exec_command}="python2"
         eval ${future_path}="${PWD}/ext/future:"
+    elif command -v /usr/libexec/platform-python >/dev/null 2>&1 ; then
+        # If a user-installed python isn't available, check for a platform-python. This is typically only used in RHEL 8.0.
+        eval ${python_exec_command}="/usr/libexec/platform-python"
```

Comment:
consider adding some logging with `echo` so that we can see in telemetry whether an install/enable occurred with the `platform-python` package and if there were any subsequent errors as a result